ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/endpoints.xml

```diff
@@ -45,14 +45,6 @@ Services (HTTP/WS) that catalog-ui-search provides for the UI
         </service-properties>
     </service>
 
-    <service ref="socketServlet" interface="javax.servlet.Servlet">
```

Comment:
We no longer register a standard servlet for websockets. The specifications pax web adheres to are for osgi's whiteboard pattern - which is for http. As a result websockets aren't directly supported through whiteboard servlet registration.